ui/app/templates/vault/cluster/secrets/backend/configuration.hbs

```diff
@@ -6,7 +6,6 @@
     path="vault.cluster.secrets.backend.list-root"
     model=this.model.id
   }}
-  @isConfigure={{true}}
```

Comment:
`isConfigure` is a param not used in the SecretListHeader component. Triple checked this and reran the test suites.